bitcoin/src/pow.rs

```diff
@@ -229,8 +230,14 @@ impl Target {
     /// [max]: Target::max
     /// [target]: crate::blockdata::block::Header::target
     #[cfg_attr(all(test, mutate), mutate)]
-    pub fn difficulty(&self) -> u128 {
-        let d = Target::MAX.0 / self.0;
+    pub fn difficulty(&self, network: Network) -> u128 {
```

Comment:
```
    /// [max]: Target::max
    /// [target]: crate::blockdata::block::Header::target
```

`L232` and `L233` document `max` and `target` params AFAICT.  These prams seem to be artifacts of the past.  I think this should be documenting the `network` param instead of `max` and `target`.

Reply:
Awesome, thanks.

Reply:
I just removed both the links.